qmetaobject/Cargo.toml

```diff
@@ -16,6 +16,7 @@ chrono_qdatetime = ["chrono"]
 [dependencies]
 qmetaobject_impl = { path = "../qmetaobject_impl", version = "=0.1.4"}
 lazy_static = "1.0"
+log = "0.4"
```

Comment:
can it be made an optional dependency?

Reply:
Then it should be documented in the crate's top level documentation in the list of features

Reply:
See https://github.com/woboq/qmetaobject-rs/issues/85#issuecomment-622176232 for my rationale.

But then again, yes, of course I could make it optional.